File: rulr/Nodes/Debug/Grid.py
import transforms3d
import rulr.Nodes
import rulr.Utils
from rulr.Utils.AutoGroup import AutoGroup
import rulr.Utils.Parameters
import numpy as np

class Node(rulr.Nodes.Base):
	def __init__(self):
		super().__init__()

		self.parameters.RoomBounds = AutoGroup()
		self.parameters.RoomBounds.Minimum = rulr.Utils.Parameters.Vector(np.array([-1, -1, 0], dtype=float))
		self.parameters.RoomBounds.Maximum = rulr.Utils.Parameters.Vector(np.array([1, 1, 0], dtype=float))

		pass

File: rulr-web/src/rulr-web/Nodes/Helpers/Grid.js
import {Base} from '../Base.js'
import * as Debug from '../../Utils/Debug.js'

class Node extends Base {
	async init() {
		await super.init();
		
		this.minor = new THREE.GridHelper(40.0, 400 / 2, 0xeeeeee, 0xeeeeee);
		this.viewportObject.add(this.minor);
	}
}
Debug.wrapClassPrototypeMethods(Node);

export { Node };